File: anthropic-cli-chatbot/node/first-call.js
import OpenAI from "openai";
import dotenv from "dotenv";

dotenv.config();

const client = new OpenAI({
  baseURL: "https://integrate.api.nvidia.com/v1",
  apiKey: process.env.NVIDIA_API_KEY,
});

async function main() {
  try {
    const response = await client.chat.completions.create({
      model: "nvidia/llama-3.3-nemotron-super-49b-v1.5",
      messages: [{ role: "user", content: "Hello! What can you do?" }],
    });

    console.log("Response:", response.choices[0].message.content);
    console.log("Usage:", response.usage);
  } catch (error) {
    console.error("API Error:", error.message);
  }
}

main();

File: anthropic-cli-chatbot/python/first_call.py
import os
from anthropic import Anthropic
from dotenv import load_dotenv

load_dotenv()

client = Anthropic()

try:
    response = client.messages.create(
        model="claude-sonnet-4-6",
        max_tokens=1024,
        messages=[{"role": "user", "content": "Hello! What can you do?"}],
    )
    print("Response:", response.content[0].text)
    print("Tokens used:", response.usage)
except Exception as e:
    print("API Error:", e)
